Sources/IndexStoreDB/IndexStoreDB.swift

```diff
@@ -80,6 +80,49 @@ public final class IndexStoreDB {
       return body(SymbolOccurrence(occur))
     }
   }
+
+  public func forEachCanonicalSymbolOccurrenceByName(symbolName: String, body: @escaping (SymbolOccurrence) -> Bool) {
```

Comment:
I suggest the name `forEachCanonicalSymbolOccurrence(byName:body:)`

Reply:
It should also return `Bool` and have `@discardableResult` like the other functions. Same with `forEachCanonicalSymbolOccurrence`.